app/models/proposal.rb

```diff
@@ -203,9 +203,6 @@ def delegate_with_default(method)
     end
   end
 
-
-  ## delegated methods ##
-
```

Comment:
Can we re-add this? I think that grouping is useful (though would be open to putting them in a mixin or something).